src/app/admin/agenda/time-slot.component.html

```diff
@@ -1,7 +1,14 @@
 <form (submit)="updateLabel()">
-  <label>
-    <input [(ngModel)]="timeSlot.label" name="label">
-  </label>
+
+  <mat-form-field class="time">
+    <input matInput [(ngModel)]="timeSlot.start" name="start" placeholder="start">
+  </mat-form-field>
+  <mat-form-field class="time">
+    <input matInput [(ngModel)]="timeSlot.end" name="end" placeholder="end">
+  </mat-form-field>
+  <p class="comment">
+    format HH:mm lub HH:mm:ss
+  </p>
```

Comment:
we should be using mat-hint for that. more info here: https://material.angular.io/components/input/overview

Reply:
i know what you mean, but i was not able to display single `mat-hint` under two input fields that are next to each other